sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (Unreleased)
 
+- Added support for service version 7.1.
```

Comment:
Perhaps there is some value talking about the fact that we do support both versions? Or has this been covered in previous updates?

```suggestion
- Added support for service version 7.1 which will now be the default version used by this package. You can target
the older 7.0 version using the `apiVersion` in the options passed to the client constructor.
```

Reply:
Sounds good, I'll think of an alternative phrasing and I'll send it to you to make it faster.

Reply:
I'm resolving this, since these files have changed considerably.